sqlglot/serde.py

```diff
@@ -45,7 +45,7 @@ def load(obj: JSON) -> Node:
     """
     if isinstance(obj, list):
         return [load(i) for i in obj]
-    if isinstance(obj, dict):
+    if isinstance(obj, dict) and "class" in obj:
```

Comment:
is this the first time we store a dict? may want to think about this

Reply:
that's correct, but cc @barakalon to double-check since iirc he made the initial implementation of serde

Reply:
while we're at this topic, please note the changes to `_norm_arg` and `hashable_args` too